src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListControl.xaml.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using AccessibilityInsights.SharedUx.ViewModels;
+using AccessibilityInsights.Extensions.Helpers;
+using AccessibilityInsights.CommonUxComponents.Dialogs;
+using AccessibilityInsights.SharedUx.FileIssue;
+using System.IO;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using AccessibilityInsights.SharedUx.Telemetry;
+using AccessibilityInsights.SharedUx.Enums;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    /// <summary>
+    /// Interaction logic for ScannerResultCustomListControl.xaml
+    /// </summary>
+    public partial class ScannerResultCustomListControl : UserControl
+    {
+        private ScannerResultCustomListContext _controlContext;
+
+        public IEnumerable ItemsSource 
+        {
+            get => lvDetails.ItemsSource;
+            set { lvDetails.ItemsSource = value; } 
+        }
+
+        public int SelectedIndex { get; set; }
+
+        public ViewBase View { get; set; }
+
+
+        /// <summary>
+        /// Keeps track of if we should automatically set lv column widths
+        /// </summary>
+        public bool HasUserResizedLvHeader { get; set; }
+
+        #region DataGridAccessibleName (Dependency Property)
+
+        public string DataGridAccessibleName
+        {
+            get { return (string)GetValue(DataGridAccessibleNameProperty); }
+            set { SetValue(DataGridAccessibleNameProperty, value); }
+        }
+
+        public static readonly DependencyProperty DataGridAccessibleNameProperty =
+            DependencyProperty.Register("DataGridAccessibleName", typeof(string), typeof(ScannerResultCustomListControl), new PropertyMetadata(null));
+
+        #endregion
+
+        public ScannerResultCustomListControl()
+        {
+            InitializeComponent();
+        }
+
+        internal void SetControlContext(ScannerResultCustomListContext controlContext)
+        {
+            _controlContext = controlContext ?? throw new ArgumentNullException(nameof(controlContext));
+        }
+
+        /// <summary>
+        /// Simulate * width for rule column
+        /// </summary>
+        /// <param name="sender"></param>
```

Comment:
While we're touching this code, let's rip out these empty placeholders. They add no value and aren't required, even on public methods